src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,95 @@
+package lotto.domain;
+
+import static lotto.enums.LottoPrize.FIRST_PLACE;
+import static lotto.enums.LottoPrize.SECOND_PLACE;
+import static lotto.enums.LottoPrize.THIRD_PLACE;
+import static lotto.enums.LottoPrize.FORTH_PLACE;
+import static lotto.enums.LottoPrize.FIFTH_PLACE;
+import static lotto.enums.LottoPrize.NO_PLACE;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lotto.enums.LottoPrize;
+
+public class Lotto {
+    private static final int LOTTO_LENGTH = 6;
+    private static final int MINIMUM_NUMBER = 1;
+    private static final int MAXIMUM_NUMBER = 45;
+    private static final String LOTTO_LENGTH_ERROR_MESSAGE = "로또 숫자는 6개 입니다.";
+    private static final String NUMBER_RANGE_ERROR_MESSAGE = "로또의 수는 1에서 45사이의 숫자 입니다.";
+    private static final String DUPLICATE_NUMBER_ERROR_MESSAGE = "로또에 중복된 수가 있습니다.";
+
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validate(numbers);
+        numbers = new ArrayList<>(numbers);
+        numbers.sort(null);
+        this.numbers = numbers;
```

Comment:
아니면 스트림을 사용하는 방법도 존재합니다!
```suggestion
        this.numbers = numbers.stream()
                .sorted()
                .toList();
```